releng/org.eclipse.xtext.sdk.target/org.eclipse.xtext.sdk.target.target

```diff
@@ -9,7 +9,7 @@
 		<unit id="org.eclipse.xtend.lib.source" version="0.0.0"/>
 		<unit id="org.eclipse.xtend.lib.macro" version="0.0.0"/>
 		<unit id="org.eclipse.xtend.lib.macro.source" version="0.0.0"/>
-		<repository location="https://ci.eclipse.org/xtext/job/xtext-lib/job/master/lastStableBuild/artifact/build/p2-repository/"/>
+		<repository location="https://ci.eclipse.org/xtext/job/xtext-lib/job/sz_issue1295/lastStableBuild/artifact/build/p2-repository/"/>
```

Comment:
Just temporarily or an accident?

Reply:
This will be reverted by means of `xtext-umbrella$ ./adjustPipelines.sh master` - this script helps to build branches across different repos against each other.